flyway-core/src/test/resources/migration/dbsupport/sqlserver/createMSDBTools.sql

```diff
@@ -0,0 +1,556 @@
+--
+-- Copyright 2010-2016 Boxfuse GmbH
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--         http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+/*
+Create Visual Studio tools in schema. Profiled from SQL Server Management studio 2014. For testing purpose only.
+ */
+IF OBJECT_ID(N'dbo.sp_upgraddiagrams') IS NULL and IS_MEMBER('db_owner') = 1
```

Comment:
They were profiled from a running MS SQL Studio instance (that is what I intended to say with the comment at the top of the file).

If you think that it could be a problem, I can also write a test script from scratch...